src/HttpKernel/ContaoKernel.php

```diff
@@ -51,10 +70,22 @@ public function registerBundles()
     /**
      * {@inheritdoc}
      */
+    public function getProjectDir()
+    {
+        if (null === $this->projectDir) {
+            $this->projectDir = dirname(dirname(dirname(dirname(dirname(__DIR__)))));
```

Comment:
With this line, the ContaoKernel now finds the path to the project dir itself. Why do we need the constructor argument then?